sklearnex/linear_model/tests/test_incremental_linear.py

```diff
@@ -46,12 +46,13 @@ def test_sklearnex_fit_on_gold_data(dataframe, queue, fit_intercept, macro_block
     inclin.fit(X_df, y_df)
 
     y_pred = inclin.predict(X_df)
+    y_pred_as_numpy = _as_numpy(y_pred)
 
-    tol = 2e-6 if y_pred.dtype == np.float32 else 1e-7
+    tol = 2e-6 if y_pred_as_numpy.dtype == np.float32 else 1e-7
```

Comment:
Similar switch to ```np_y_pred``` as in another PR recommendation

Reply:
Already commented it on other PR, lets keep discussion there.